lib/generators/boring/pry_byebug/install_generator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Boring
+  module PryByebug
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds pry-byebug gem to the application for step-by-step debugging and stack navigation capabilities to pry using byebug"
+
+      def add_pry_byebug_gem
+        say "Adding pry_byebug gem", :green
+
+        gem_content = <<~RUBY
```

Comment:
We can use `bundle add pry-byebug` which should actually be preferred because it adds gem with latest stable version. We don't have to do bundle install after that.

So this method can only include:

`bundle add pry-byebug`